### ISSUE TITLE
[YouTube] Unable to look up account info: HTTP Error 400: Bad Request

### ISSUE THREAD

**Issue by kaziusj** [ytdl-org/youtube-dl#32174]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.12.17** 
       I use the most recent downloadable windows exe
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
       please see description that this is no copy of https://github.com/ytdl-org/youtube-dl/issues/31530 
- [x] I've read bugs section in FAQ

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-u', 'PRIVATE', '-i', '-o', 'C:\\Users\\yzaak\\Music\\AggroPhonk\\5redo\\%(title)s.%(ext)s', '-x', '--audio-format', 'mp3', '--audio-quality', '0', '--add-metadata', 'https://www.youtube.com/playlist?list=PLL8J5fRjnIyp9GG7kCGQEiQ-WVfTq3__Q', '-v']
Type account password and press [Return]:
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.22621
[debug] exe versions: ffmpeg 3.3.2, ffprobe 3.3.2
[debug] Proxy map: {}
[youtube:tab] Downloading login page
[youtube:tab] Looking up account info
WARNING: Unable to look up account info: HTTP Error 400: Bad Request
[youtube:tab] PLL8J5fRjnIyp9GG7kCGQEiQ-WVfTq3__Q: Downloading webpage
[download] Downloading playlist: 5redo
[youtube:tab] playlist 5redo: Downloading 6 videos
[download] Downloading video 1 of 6
[youtube] Downloading login page
[youtube] Looking up account info
WARNING: Unable to look up account info: HTTP Error 400: Bad Request
[youtube] j1zkJJ6-Ja0: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[download] Downloading video 2 of 6
[youtube] K6yQ7JTGp00: Downloading webpage
[youtube] K6yQ7JTGp00: Downloading player 65ceadf9
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[download] Downloading video 3 of 6
[youtube] YZ5Yo8oXnpg: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[download] Downloading video 4 of 6
[youtube] x58PcUOeYpU: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[download] Downloading video 5 of 6
[youtube] bCQqPUuxyKY: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[download] Downloading video 6 of 6
[youtube] FNgGIcwr-Pk: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

[download] Finished downloading playlist: 5redo
```

## Description

Only for videos that contain a '-' symbol in their title do i get this error message (30+ other videos without '-' in their title went well, in an earlier attempt). The current playlist (5redo) contains the remaining 6 videos that gave me my error, and all have the '-' in their title. As noted, all 6 URLs are valid.

I hope i made no mistakes.. happy squashing!

**Comment by SK86TUD** [comment]
For me the problem occurres also when there is no '-' in the URL, e.g., https://www.youtube.com/watch?v=kwJTpJjLJi4

**Comment by kaziusj** [comment]
your title says "motorhead - enter sandman"
all my failed titles have the '-' while all titles i tried without '-' worked

**Comment by SK86TUD** [comment]
true - I thought in the URL - but you mean the title on youtube. I see.

**Comment by dirkf** [comment]
I couldn't reproduce this with the release (2021.12.17) version: it failed signature extraction.

I tried a similar command without `-u ...` using the latest code.
<details><summary>The log shows it appearing to work:</summary>

```console
$ python -m youtube_dl -v -f worst --test  'https://www.youtube.com/playlist?list=PLL8J5fRjnIyp9GG7kCGQEiQ-WVfTq3__Q'
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-f', u'worst', u'--test', u'https://www.youtube.com/playlist?list=PLL8J5fRjnIyp9GG7kCGQEiQ-WVfTq3__Q']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: d0b1f995c
[debug] Python version 2.7.18 (CPython) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[youtube:tab] PLL8J5fRjnIyp9GG7kCGQEiQ-WVfTq3__Q: Downloading webpage
[download] Downloading playlist: 5redo
[youtube:tab] playlist 5redo: Downloading 6 videos
[download] Downloading video 1 of 6
[youtube] j1zkJJ6-Ja0: Downloading webpage
[debug] [youtube] Decrypted nsig MhpM7bx3eqsCpDTEP => FH7Iqa-jMK3ndQ
[debug] [youtube] Decrypted nsig ELzif9KQYWdKYiC99 => VTLeXVxrxIEREQ
[debug] Invoking downloader on u'https://rr4---sn-cu-aigss.googlevideo.com/videoplayback?sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Csvpuc%2Cmime%2Cns%2Ccnr%2Cratebypass%2Cdur%2Clmt&fexp=24007246&ei=6s5bZIqAGISSp-oP35a7-As&ip=46.208.6.24&id=o-AP1XbcQG0XTpQIc3q1NOcuK4QaalyLaCBfsPy7QvSUyE&txp=5438434&svpuc=1&cnr=14&requiressl=yes&ratebypass=yes&source=youtube&sig=AOq0QJ8wRAIgd3MBy77eoogodv3Lqrsd35m7nMyhukxUZrJ2clFZ3rsCIH_UuxQ9mLInANBPoueCoXlz2La23-yLsOnLYBUg6zlN&dur=174.265&ns=l3rRlrFv69H4_kcq4Q1dwDoN&initcwndbps=1503750&vprv=1&lsig=AG3C_xAwRQIhAM-qIzTy67uokEmKZ2AOeN18tl2C5sv9GFnj1GGq5WfRAiA5lcdwlBJHhM7CW1o1sZioVwaeuXmJZFZG_e4pROcVPQ%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lmt=1662790297852833&c=WEB&expire=1683759946&mime=video%2Fmp4&fvip=3&itag=18&mm=31%2C29&mn=sn-cu-aigss%2Csn-cu-c9i6&mh=Gx&n=VTLeXVxrxIEREQ&mt=1683737837&mv=m&pl=25&ms=au%2Crdu&mvi=4'
[download] Destination: 509 $ICARIO - DEADLY HEIST-j1zkJJ6-Ja0.mp4
[download] 100% of 10.00KiB in 00:00
[download] Downloading video 2 of 6
[youtube] K6yQ7JTGp00: Downloading webpage
[debug] [youtube] Decrypted nsig w60xMBgHZYpv8C8hz => ExUt3rJcJvSRUA
[debug] [youtube] Decrypted nsig W96lAhSWZaPndMwuN => GdghgPO47VG3vw
[debug] Invoking downloader on u'https://rr1---sn-cu-aigss.googlevideo.com/videoplayback?sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Csvpuc%2Cmime%2Cns%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&fexp=24007246&ei=785bZJzGMtfOxwLQuISoDA&ip=46.208.6.24&clen=9546676&id=o-AFOcgWV0Y7UKLO4Fxhyu2F40BQhMqGLOJZNh4dXx6xiC&txp=6319224&svpuc=1&gir=yes&requiressl=yes&ratebypass=yes&source=youtube&sig=AOq0QJ8wRQIgFlqE1Q-QUF_21RkdmRUgXZnR_oCp7KWWWQbzwI0wpE0CIQChg_ZOFuvp8794oR2AZsyWmG0w-bDiPDE9pqpFj_co7w%3D%3D&dur=127.013&ns=ZZl-LT9w1tGMWpsrlZ4whNEN&initcwndbps=1436250&vprv=1&lsig=AG3C_xAwRgIhAIPRnzL7VTSeBRMTzWBWDMn0PjVbGC9Hj0Vgvn0cSMa1AiEA2Mc_aBDi1c0jRnOrtQkgjpyk7REMibevQyofwrwmhTA%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lmt=1671190212304540&c=WEB&expire=1683759951&mime=video%2Fmp4&fvip=4&itag=18&mm=31%2C29&mn=sn-cu-aigss%2Csn-cu-c9i6&mh=Wm&n=GdghgPO47VG3vw&mt=1683738074&mv=m&pl=25&ms=au%2Crdu&mvi=1'
[download] Destination: SHADXWBXRN - KNIGHT II (MUSIC VIDEO)-K6yQ7JTGp00.mp4
[download] 100% of 10.00KiB in 00:00
[download] Downloading video 3 of 6
[youtube] YZ5Yo8oXnpg: Downloading webpage
[debug] [youtube] Decrypted nsig ok2JxObrvhUgaXAfO => qq-F1TLZcECk3A
[debug] [youtube] Decrypted nsig TSC354uwH1rQ6uvpu => lAy_bOHxL2LIpA
[debug] Invoking downloader on u'https://rr4---sn-cu-aigss.googlevideo.com/videoplayback?sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Csvpuc%2Cmime%2Cns%2Ccnr%2Cratebypass%2Cdur%2Clmt&fexp=24007246&ei=9c5bZMvgKIu11ga0xbSgCg&ip=46.208.6.24&id=o-ACGeFPBAhYvjIlr7h_jsC61afVfUqOmQWVGM3OgfG6hy&txp=6318224&svpuc=1&cnr=14&requiressl=yes&ratebypass=yes&source=youtube&sig=AOq0QJ8wRQIhAK1c2Mr01sWwdWyvjoPLKhcsuXWFgIf421ZRltl8VbFhAiBMcufUoE9uPK5uACu3SR31EAsUamVc4P87ROjEc1rz1Q%3D%3D&pcm2cms=yes&dur=129.126&ns=RnxusKzXU7GKLy7zr7gKSA4N&initcwndbps=1436250&vprv=1&lsig=AG3C_xAwRgIhAPytfJkGC-5pf-dOyjNLL77wnqoKYzeWWOgceQ_gB0a_AiEAvYFPGuof24ZUUB9wXQTUtY_tcklGWyXsAOkuDOUIed0%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lmt=1658973294080547&c=WEB&expire=1683759957&mime=video%2Fmp4&fvip=5&itag=18&mm=31%2C29&mn=sn-cu-aigss%2Csn-cu-c9id&mh=tN&n=lAy_bOHxL2LIpA&mt=1683738074&mv=m&pl=25&ms=au%2Crdu&mvi=4'
[download] Destination: SHADXWBXRN - COURAGE-YZ5Yo8oXnpg.mp4
[download] 100% of 10.00KiB in 00:00
[download] Downloading video 4 of 6
[youtube] x58PcUOeYpU: Downloading webpage
[debug] [youtube] Decrypted nsig 2lpjMLGBipgzl55Kn => 5fZfwoWgAaPiRw
[debug] [youtube] Decrypted nsig utdobEEE9vA-gSqXY => X2zkJJRHUb4XPw
[debug] Invoking downloader on u'https://rr2---sn-cu-aigss.googlevideo.com/videoplayback?sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Csvpuc%2Cmime%2Cns%2Ccnr%2Cratebypass%2Cdur%2Clmt&fexp=24007246&ei=-85bZLS5I7GPxN8PhfWYsAI&ip=46.208.6.24&id=o-AMe6YGDWcByaQM85gqhOupSkX_sSVyg6gH0wNslVMuO8&txp=6318224&svpuc=1&cnr=14&requiressl=yes&ratebypass=yes&source=youtube&sig=AOq0QJ8wRQIgR3eL_z3Mnp_an2bpRpphrsfWb-4cD0B8QQ12BPFYBe4CIQDVEj--Ugm88BK3y8J-DPATMV3183thkbDwSVIyXjwrMw%3D%3D&dur=139.482&ns=9Y8YdT6HMl-ki0-KznkvfyUN&initcwndbps=1436250&vprv=1&lsig=AG3C_xAwRgIhAJHFwh7vGVo-Re0iE0Qy3cyKFkao6wuDGDbRUvy_a4UnAiEA2TY1GcCSDVJyliX9a7yNP07gBEkpX0eeXAI0Osp_dk8%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lmt=1661745633293974&c=WEB&expire=1683759963&mime=video%2Fmp4&fvip=2&itag=18&mm=31%2C29&mn=sn-cu-aigss%2Csn-cu-c9id&mh=NH&n=X2zkJJRHUb4XPw&mt=1683738074&mv=m&pl=25&ms=au%2Crdu&mvi=2'
[download] Destination: SHADXWBXRN - WARLOCK-x58PcUOeYpU.mp4
[download] 100% of 10.00KiB in 00:00
[download] Downloading video 5 of 6
[youtube] bCQqPUuxyKY: Downloading webpage
[debug] [youtube] Decrypted nsig ks_cISz1g6NrDPoSn => p1X8EHc85-iUGA
[debug] [youtube] Decrypted nsig NJf-D4fmS_aq3Lptq => 7aGWfIE7u5Yzsw
[debug] Invoking downloader on u'https://rr2---sn-cu-aigss.googlevideo.com/videoplayback?sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Csvpuc%2Cmime%2Cns%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&fexp=24007246&ei=Ac9bZIyBIvGrxN8Pk8ym8Aw&ip=46.208.6.24&clen=12629558&id=o-AD7gdAof_z_b3llPLn0VLwCUxoJqGXNTViUckjz5jt2t&txp=5538434&svpuc=1&gir=yes&requiressl=yes&ratebypass=yes&source=youtube&sig=AOq0QJ8wRAIgat-sShiACTZdM60FFf18Vwce3ihkE2gRn4tfG2h7VkQCIEAV5fTFjwWnKpkA3s_6G7tN_NrZfircXsjvyIoRPKpB&dur=230.992&ns=0j2qSBAUdlOmgDnYQK_EVAAN&initcwndbps=1436250&vprv=1&lsig=AG3C_xAwRgIhAOErl0E0gmtfUh5z4InufBxogmCVMesyuwd15gIZ1TXYAiEAkdxdlghzWEI6IKpfY5GU6kEMO9wr9xnMudxHyEROuwg%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lmt=1674587741188158&c=WEB&expire=1683759969&mime=video%2Fmp4&fvip=4&itag=18&mm=31%2C29&mn=sn-cu-aigss%2Csn-cu-c9i6&mh=vo&n=7aGWfIE7u5Yzsw&mt=1683738074&mv=m&pl=25&ms=au%2Crdu&mvi=2'
[download] Destination: phonk.me, KIIXSHI - GHOST!-bCQqPUuxyKY.mp4
[download] 100% of 10.00KiB in 00:00
[download] Downloading video 6 of 6
[youtube] FNgGIcwr-Pk: Downloading webpage
[debug] [youtube] Decrypted nsig gd5V_-sGhqk2uzswi => owURiL3LEBGJXg
[debug] [youtube] Decrypted nsig IG6s4oKpPSOPVWlw5 => 7EBoiEuw6u_6WA
[debug] Invoking downloader on u'https://rr1---sn-cu-aigss.googlevideo.com/videoplayback?sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Csvpuc%2Cmime%2Cns%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&fexp=24007246&ei=Bs9bZOelOoaQp-oP0v2I-AU&ip=46.208.6.24&clen=5059223&id=o-ANcmoadNTzBc6TZztaYwAsJOZVp_uVAm6QkwHvQcmKLs&txp=5438434&svpuc=1&gir=yes&requiressl=yes&ratebypass=yes&source=youtube&sig=AOq0QJ8wRgIhAO8tot6BulCLNpIzK76jqXbcJ4iHw40HmdQ7hrI189rhAiEA8rT2JUEBN2APtJlOtAyK6uRJMJylRMlSWIXS4k-VIw8%3D&dur=212.625&ns=RFCBe3Bj3cpTthHYq82X_uUN&initcwndbps=1436250&vprv=1&lsig=AG3C_xAwRAIgJdueWPHwp0Weo1-68E1vSAfEl3V-BtLXgihmo-vUpPICIHfFwxNvDuNaBQRWvwBq5Lcng-zKSAuR1YOsW5FfSbKg&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lmt=1676061254631557&c=WEB&expire=1683759975&mime=video%2Fmp4&fvip=4&itag=18&mm=31%2C29&mn=sn-cu-aigss%2Csn-cu-c9i6&mh=LV&n=7EBoiEuw6u_6WA&mt=1683738074&mv=m&pl=25&ms=au%2Crdu&mvi=1'
[download] Destination: MoonDeity x Phonk Killer - DEATH DAGGER-FNgGIcwr-Pk.mp4
[download] 100% of 10.00KiB in 00:00
[download] Finished downloading playlist: 5redo
$
```

</details>

Please update to master code, eg using one of the non-yt-dlp solutions suggested in #31350. Then try the command again with and without `-u ...`. If login **is** required and the login procedure fails, please collect cookies from a logged-in browser session and pass them with `--cookies ...` instead of `-u ...` (see FAQ).

@Jelfens, your URL is wrong (video id is too short) and **not "quoted"**.

**Comment by stefan-reich** [comment]
@dirkf Are you saying there is a version that fixes this? The bug is real, I can confirm. And why would a browser session be needed?

**Comment by silentswrd** [comment]
I'm having this issue as well trying to run `youtube-dl https://youtu.be/t7V_TA-pk2U`.

**Comment by silentswrd** [comment]
**I found a fix:**

1. `brew update`
2. `brew reinstall youtube-dl`

This worked for me, let me know if it works for anyone else.

**Comment by dirkf** [comment]
Have you (a) looked at the log that I posted (b) updated your yt-dl (which you need to do anyway because of #31530, and other issues) (c) carried out the tests that I suggested?

Specifying cookies from a logged-in browser session (where the target plays correctly) is a standard and more reliable (though more painful) alternative to specifying username and password, if your target videos need authentication: they didn't for me:
````
$ ls *.mp4
509 $ICARIO - DEADLY HEIST-j1zkJJ6-Ja0.mp4
MoonDeity x Phonk Killer - DEATH DAGGER-FNgGIcwr-Pk.mp4
phonk.me, KIIXSHI - GHOST!-bCQqPUuxyKY.mp4
SHADXWBXRN - COURAGE-YZ5Yo8oXnpg.mp4
SHADXWBXRN - KNIGHT II (MUSIC VIDEO)-K6yQ7JTGp00.mp4
SHADXWBXRN - WARLOCK-x58PcUOeYpU.mp4
$
```

**Comment by kaziusj** [comment]
@dirkf you clearly get different feedback and i am confused. As noted, i'm on windows and (afaik) i downloaded the most recent exe from ttps://github.com/ytdl-org/youtube-dl (via the link in the installation section). Is this .exe not the latest version? 

Related to this question: using the exe I also just tried downloading one of these videos without specifying a username, but the error stayed identical. (Unable to extract uploader id..  )

PS Maybe a detail: you download mp4s, while i (when yt-dl works) download a .webm and convert it to .mp3
PPS atm i have no idea how to build a windows exe from master :/

**Comment by entr0pnia** [comment]
having the exact same issue as @kaziusj and have the latest exe

**Comment by kaziusj** [comment]
Sorry @dirkf i missed your first comment, let me rephrase with a b and c:
a) your logs are different from mine (whether i use -u or not), probably either because you somehow use a different version or because you extract .mp4 instead of the .webm
b) like @entr0pnia, i have the most recent windows exe, but that seems to behave differently from your most recent master version
c) "If login is required and the login procedure fails, please collect cookies from a logged-in browser session and pass them with --cookies ... instead of -u ... (see FAQ)." login does not seem to be required to get the same error at least, but ill look into it later today. . TBC

**Comment by dirkf** [comment]
>...Is this .exe not the latest version?

As you mentioned it, I assumed you had actually read #31530. Do that first. As I clearly stated, my log is from the development master. Possibly because of different crypto Python modules bundled into the Windows .exe, the release version fails well ahead of any error in your log.

@entropia, which "same issue"? If it's just the #31530 issue, look there.

**Comment by kaziusj** [comment]
@dirkf sorry, yesterday i actually read your answer in #31530 but then i somehow wasnt able to find the yt-dlp.exe then but only the master build (which i thought i was unable to use). I just tried that yt-dlp.exe and it works, thanks!

In my defense, i did state that i used the windows exe and i did post my problem here (and not as an issue at this yt-dlp project). Perhaps future confusion can ben prevented when the logs shows more details than "youtube-dl version 2021.12.17".

**Comment by dirkf** [comment]
There is a nightly build of yt-dl for Windows from a fork, linked from #31350.
